MatrixSDK/Crypto/Algorithms/Megolm/MXMegolmEncryption.m

```diff
@@ -299,6 +300,21 @@ - (MXHTTPOperation *)ensureOutboundSession:(MXUsersDevicesMap<MXDeviceInfo *> *)
     session.shareOperation = [self shareKey:session withDevices:shareMap success:^{
 
         session.shareOperation = nil;
+        if (shareMap.count)
```

Comment:
Why do we need to do it here?
It is done too in `shareKey()`.

Reply:
Indeed ... Good catch !